components/camel-cmis/src/main/java/org/apache/camel/component/cmis/CamelCMISConstants.java

```diff
@@ -19,9 +19,15 @@
 public interface CamelCMISConstants {
     String CMIS_DOCUMENT = "cmis:document";
     String CMIS_FOLDER = "cmis:folder";
+    String CMIS_ACTION = "cmis:action";
     String CMIS_FOLDER_PATH = "CamelCMISFolderPath";
+    String CMIS_OBJECT_ID = "ObjectId";
```

Comment:
Could be very common name and intersect with another/custom/third-party component or customer route header name.